packages/SwingSet/src/kernel/vatTranslator.js

```diff
@@ -175,11 +175,12 @@ function makeTranslateKernelDeliveryToVatDelivery(vatID, kernelKeeper) {
     return startVatMessageVatDelivery;
   }
 
-  /** @type { import('../types-external.js').VatDeliveryStopVat } */
-  const stopVatMessage = harden(['stopVat']);
-
-  function translateStopVat() {
-    return stopVatMessage;
+  /**
+   * @param { SwingSetCapData } disconnectObject
+   * @returns { import('../types-external.js').VatDeliveryStopVat }
+   */
+  function translateStopVat(disconnectObject) {
```

Comment:
Let's name this `disconnectObjectCD` or `disconnectObjectCapData` to match the name we use in `kernel.js` where we build the "stopVat" `VatDeliveryObject`, and distinguish it from the variables named `disconnectObject` which hold non-capdata.